### PR TITLE
Added changes in contributing.md regarding the make command and fixed…

### PR DESCRIPTION
**Pull request by prashant-isbest**
… the naming of the checkout branch from which pull request will be send

Signed-off-by: Prashant Joshi <scienceprashant27@gmail.com>

**Description**

This PR fixes # contributing.md

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ yes] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 07fb442d37271dc630cbbe9c2f86f59a400d7b06 at: https://614f511fa494f69311915bf3--layer5ng.netlify.app

**Review by debo19 — APPROVED** [review]
LGTM 🎉

**Review by warunicorn19 — APPROVED** [review]
Thank you @prashant-isbest.😄

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -80,17 +80,18 @@ Please contribute! The Layer5 site uses Gatsby. The process of contributing to d
 1. Get a local clone of your fork of the site.
 `git clone https://github.com/layer5io/layer5`
 1. Switch to the `master` branch.
-`git checkout master`
+`git switch master`
```
Comment (Nikhil-Ladha) [inline CONTRIBUTING.md:83]: Why this change?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -80,17 +80,18 @@ Please contribute! The Layer5 site uses Gatsby. The process of contributing to d
...
 1. Create and checkout a new branch to make changes within
-`git checkout -b <my-username>/<my-changes>`
+`git checkout -b <my-changes-branchname>`
 1. Edit site/add content.
 `vi <specific page>.md` # or use your favorite IDE
 1. Run site locally to preview changes.
 `make site` # this will run a local web server with "live reload" conveniently enabled.
+(NOTE make is a linux command , if you are using windows you can install wsl , and then run command `sudo apt install make` )
```
Comment (Nikhil-Ladha) [inline CONTRIBUTING.md:90]: I doubt that!
`make` do work in windows as well.
 We have GNU make for Windows.

File: CONTRIBUTING.md (hunk 1 above)
Comment (warunicorn19) [reply CONTRIBUTING.md:83]: @Nikhil-Ladha actually if we don't do `git switch master` the previous commits of a different branch also get included into the new one.

Comment (Nikhil-Ladha) [reply CONTRIBUTING.md:83]: Which branch and what commits? Example?

Comment (warunicorn19) [reply CONTRIBUTING.md:83]: For example, if I'm already working on this repo and I create a branch to work on some issue, I do the changes and make a PR, meanwhile I have to work on another issue and so I create another branch to work on, before creating a new branch if I don't do `git switch master` all my changes from the previous branch would also come onto the new branch.

Comment (warunicorn19) [reply CONTRIBUTING.md:83]: Though ig we would probably not need this information here considering this is for getting started, but then we could also get rid of the `git checkout master` here cause after cloning we would be on the master branch itself 🤔, right? @Nikhil-Ladha

Comment (Nikhil-Ladha) [reply CONTRIBUTING.md:83]: That's obvious.
But, the same can be done with `git checkout master`

Comment (warunicorn19) [reply CONTRIBUTING.md:83]: Oh, I didn't know that😅, I've actually always done `git switch master` just to be safe.

Comment (Nikhil-Ladha) [reply CONTRIBUTING.md:83]: Ok, so the thing is `git checkout master` does exactly everything of `git switch master` in fact, git` switch` is a (code)copy of `git checkout master`. `git switch` was added considering the dual nature of `git checkout` to create branches and restore changes. And, therefore to avoid confusions 2 new commands `git switch` and `git restore` was added.

Comment (Nikhil-Ladha) [reply CONTRIBUTING.md:83]: > Oh, I didn't know thatsweat_smile, I've actually always done git switch master just to be safe.
It's fine. Just, that the such PRs should have a better commit-msg and description to justify the changes being done.

Comment (warunicorn19) [reply CONTRIBUTING.md:83]: Aah okay, makes sense, yes you're right.
